pkg/dashboards/types.go

```diff
@@ -258,13 +287,8 @@ type DashboardPageInput struct {
 	GUID common.EntityGUID `json:"guid,omitempty"`
 	// The name of the page.
 	Name string `json:"name"`
-
-	// NOTE: The JSON description of the following attribute, "Widgets" has been modified manually
-	// (removal of "omitempty") to facilitate creating pages with no widgets (empty pages).
-	// Please DO NOT regenerate/modify this attribute and its datatype via Tutone (which would add "omitempty" back).
-
 	// A nested block of all widgets belonging to the page.
-	Widgets []DashboardWidgetInput `json:"widgets"`
+	Widgets []DashboardWidgetInput `json:"widgets,omitempty"`
```

Comment:
Can you please revert this change (considering the note in the comments above this line)

This change was made in an older PR in order to remove `omitempty` against widgets, as this field would get omitted from the request passed to the API if no widgets are included in the page that is added in the API request, thereby blocking users from creating dashboards with empty pages. This had to be added manually and not via Tutone, since Tutone currently doesn't support removing `omitempty` against optional fields.

Reply:
same here